generated/nodejs/test/errorreporting.js

```diff
@@ -146,11 +161,11 @@ function listErrorGroupStats() {
       }).on('end', function() {
         var firstErrorGroupId = groups[0].group.group_id;
```

Comment:
I guess I asked you about this. On ruby, for listGroupStats, since it returns an array, I can call eash_with_index and use index 0 to find out the first group's ide, but on node, with stream.on('data'), I don't know how to get the first group since I don't a way to use a index, so I just pushed all the groups into an array, so at stream.on('end'), I can use group[0] to get the first group's id.